ext/mysql2/mysql_enc_name_to_ruby.h

```diff
@@ -137,7 +137,7 @@ mysql2_mysql_enc_name_to_rb (str, len)
       {""},
       {"dec8", NULL},
       {"cp850", "CP850"},
-      {"latin1", "ISO-8859-1"},
```

Comment:
I had to resolve a merge conflict here (some of the entries in this list were reordered)